pypit/arms.py

```diff
@@ -62,6 +64,9 @@ def ARMS(spectrograph, fitstbl, setup_dict):
     # Init calib dict
     calib_dict = {}
 
+    # Spectrometer class
+    # This will be passed into the Calibrations class in PR #368
+    spectro_class = spectro_utils.load_spec_class(spectrograph=spectrograph)
```

Comment:
Also, I think spectro_class can be passed into `ARMS` instead of spectrograph. If we want the name of the spectrograph we should just be able to use a call like: `spectro_class.spectrograph`.

My general comment (not just on this line) is that I don't think we need to pass both `spectro_class` and `spectrograph` around.

Reply:
Agreed.  Would it be ok if I defer this to a future PR (maybe even just the current Calibrations one)?

I will otherwise be just as quickly deleting what I generate.